src/test/omap_bench.cc

```diff
@@ -84,9 +84,9 @@ int OmapBench::setup(int argc, const char** argv) {
       	   << "rand for random sizes between\n"
       	   << "                        0 and max size, uniform for all sizes"
       	   << " to be specified size.\n"
-      	   << "                        (default "<<value_size;
-      cout <<"\n  --name          the rados id to use (default "<<rados_id;
-      cout<<")\n";
+           << "                        (default uniform)\n";
+      cout << "	--name          the rados id to use (default "<<rados_id
```

Comment:
>  I didn't add the space just want to keep consistent with others above it, like "threads", "objests", etc.
please take this chance to add the space since you are change this line anyway.